addons/l10n_mz/data/account_tax_template_data.xml

```diff
@@ -0,0 +1,358 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="account.tax.template" id="vat_sale_16">
+        <field name="name">16%</field>
+        <field name="description">16%</field>
+        <field name="amount" eval="16"/>
+        <field name="amount_type">percent</field>
+        <field name="chart_template_id" ref="l10n_mz_chart_template"/>
+        <field name="type_tax_use">sale</field>
+        <field name="tax_group_id" ref="tax_group_vat_16"/>
+        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
+                (0,0, {
```

Comment:
```suggestion
                (0, 0, {
```
I'm okay with both but keep it consistent :)

(This applies other places in this file)